### PR TITLE
Reaction network updates

### PR DESCRIPTION
**Pull request by semiller10**
This PR includes a few fixes.
- Contigs database migration from `v21` to `v22` fixes the failure to initialize three required metavariables used in reaction network storage when a `v21` database was generated from scratch
- Proper case of default `misc` directory for reference database installation
- Fixed run info messages reporting reference database paths

### CODE REVIEW

File: anvio/biochemistry/reactionnetwork.py [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ class KODatabase:
     Unless an alternative directory is provided, the database is downloaded and set up in a
     default anvi'o data directory, and loaded from this directory in network construction.
     """
-    default_dir = os.path.join(os.path.dirname(ANVIO_PATH), 'data/MISC/KEGG/KO_REACTION_NETWORK')
+    default_dir = os.path.join(os.path.dirname(ANVIO_PATH), 'data/misc/KEGG/KO_REACTION_NETWORK')
```
Comment (meren) [inline anvio/biochemistry/reactionnetwork.py:1083]: I corrected this but this seems to be going back to what it was, @semiller10

Comment (semiller10) [reply anvio/biochemistry/reactionnetwork.py:1083]: I looked at it again, and `misc` should be lowercase. The error was not there for KEGG, but for ModelSEED, where I had it uppercase.

Comment (semiller10) [reply anvio/biochemistry/reactionnetwork.py:1083]: @meren See this issue https://github.com/merenlab/anvio/issues/2142#issuecomment-1761684115

Comment (meren) [reply anvio/biochemistry/reactionnetwork.py:1083]: So all 'misc' in that file are lowercase now? Sorry I'm very confused :)

Comment (semiller10) [reply anvio/biochemistry/reactionnetwork.py:1083]: So was I :) Turns out it's lowercase throughout the codebase.

**Pull request merged**